Cargo.toml

```diff
@@ -8,7 +8,8 @@ license = "MIT"
 [dependencies]
 codec = { package = "parity-scale-codec", version = "3.1.2", default-features = false, features = ["derive", "full"] }
 dao-io = { path = "io" }
-ft-io = { git = "https://github.com/gear-dapps/fungible-token.git", tag = "0.1.0" }
+ft-main-io = { git = "https://github.com/gear-dapps/sharded-fungible-token.git" }
+ft-logic-io = { git = "https://github.com/gear-dapps/sharded-fungible-token.git" }
 gstd = { git = "https://github.com/gear-tech/gear.git", features = ["debug"] }
 primitive-types = { version = "0.11.1", default-features = false, features = ["scale-info"]}
```

Comment:
Please remove `primitive-types` if it is not used. And from the `dao-io` crate too.